independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/codegen/buildtool/GradleGenerator.java

```diff
@@ -20,7 +20,7 @@
 import java.util.concurrent.atomic.AtomicReference;
 import java.util.function.Consumer;
 
-public final class GradleBuildFilesCreator {
+public final class GradleGenerator {
```

Comment:
I am not sure why I renamed it, I will rename it back to the original name later